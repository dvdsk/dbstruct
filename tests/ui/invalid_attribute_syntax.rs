use dbstruct::dbstruct;

#[dbstruct("db"=sled)]
struct Test {
    field: Option<u8>,
}

fn main() {}
