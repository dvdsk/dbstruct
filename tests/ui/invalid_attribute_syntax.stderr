error: invalid syntax for dbstruct option

         = help: the option should be a single word not enclosed in "

 --> tests/ui/invalid_attribute_syntax.rs:3:12
  |
3 | #[dbstruct("db"=sled)]
  |            ^^^^
